docs/GettingStartedDocs/Contributors/CodeCoverage.md

```diff
@@ -4,7 +4,7 @@ This document shows how to get the code coverage of OE.
 The code coverage analysis is based on [Gcov](https://gcc.gnu.org/onlinedocs/gcc/Gcov.html), a source-based coverage implementation provided by GNU GCC.
 Currently, both Clang and GCC supports Gcov.
 
-*Note*: The current implementation of OE only supports clang-7.
+*Note*: The current implementation of OE only supports clang-8.
```

Comment:
This might not be true unless we update the libgcov, which is currently based on clang-7. Given that the code coverage is currently disabled, we can update this later.

Reply:
Fixed.